packages/checkout/sdk/src/env/constants.ts

```diff
@@ -375,8 +375,8 @@ export const ERC721ABI = [
 ];
 
 // Gas overrides -- Anti-spam mechanism for when the baseFee drops low
-// https://github.com/immutable/imx-docs/blob/main/docs/main/zkEVM/overview/gas-configuration.md
+// https://docs.immutable.com/docs/zkEVM/architecture/gas-config
 export const GAS_OVERRIDES = {
-  maxFeePerGas: BigNumber.from(102e9),
-  maxPriorityFeePerGas: BigNumber.from(101e9),
+  maxFeePerGas: BigNumber.from(15e9),
+  maxPriorityFeePerGas: BigNumber.from(10e9),
```

Comment:
new values, confirmed that this are good to go

Reply:
Can you extract these as constants and give them a name like ZKEVM_MAX_FEE_PER_GAS etc...

Reply:
perhaps rename GAS_OVERRIDES to IMMUTABLE_ZKVEM_GAS_OVERRIDES